src/components/SearchComponent/components/searchValues.js

```diff
@@ -7,12 +7,12 @@ export const SearchValues = ({stocks, deleteStock}) => {
         <div>
             {stocks.map( stock => {
                 return (
-                    <div key={stock}>
+                    <span style={{float: "left"}} key={stock}>
```

Comment:
I'm not too sure about this. Lets discuss this during our next standup. I have an idea for adding a display:inline-block to the container div instead.

Reply:
are you stupid

Reply:
maybe but I'm not float:left stupid.